src/dashboard/src/components/api/views.py

```diff
@@ -80,18 +80,18 @@ def wrapper(request, *args, **kwargs):
 def authenticate_request(request):
     error = None
 
+    client_ip = request.META['REMOTE_ADDR']
+    whitelist = helpers.get_setting('api_whitelist', '').split()
+    if client_ip in whitelist:
+        LOGGER.debug('API called by trusted IP %s', client_ip)
+        return None
+
```

Comment:
Yes, you described the change much better than I did. It needs to be that way because we're not going to know in advance what's the IP address of the client, not in AWS ECS. I also changed the default value of the whitelist to be an empty string to give everyone an opportunity to consider if whitelisting 127.0.0.1 is ok.

I think I'm going to create a migration that updates the whitelist setting to empty string if its value was 127.0.0.1.